enhancements/cloud-integration/tokenized-auth-enablement-operators-on-cloud.md

```diff
@@ -0,0 +1,340 @@
+---
+title: tokenized-auth-enablement-operators-on-cloud
+authors:
+  - "@bentito"
+  - "@jharrington22"
+  - "@gallettilance"
+reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
+  - "@joelanford" # for OLM, OperatorHub changes
+  - "@bparees" # for OLM, OperatorHub changes
+  - "@csrwng" # for HyperShift changes
+  - "@spadgett" # for Console changes
+  - "@abutcher" # for CCO changes
+approvers:
+  - "@sdodson"
+  - "@joelanford"
+  - "@bparees"
+api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
+  - "@JoelSpeed " # for CredentialsRequest changes
+creation-date: 2023-01-21
+last-updated: 2023-04-19
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - https://issues.redhat.com/browse/OCPBU-4
+  - https://issues.redhat.com/browse/PORTENABLE-471
+  - https://issues.redhat.com/browse/CCO-286
+see-also:
+  - []
+replaces:
+  - []
+superseded-by:
+  - []
+---
+
+# Tokenized Authentication for Red Hat Operators on Cloud Providers
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Many Cloud Providers offer services that allow authentication via Temporary Access Tokens (TAT).
+
+For example, Amazon Web Services (AWS) Secure Token Service (STS) allows you to request temporary limited-privilege
+credentials. A concrete example is write permission on an S3 bucket on AWS. STS is supported on all the major cloud
+providers where OpenShift is deployed.
+
+Currently, several Red Hat operators provide detailed instructions for how to use the operator on clusters where CCO is
+in Manual Mode. The process is designed to have manual operations performed by the customer that are inconsistent across
+layered product operators.
```

Comment:
This sentence reads "the process is designed ... to be inconsistent across layered product operators". Is that what you intended to say?

Reply:
What its trying to convey here is that the manual operations required to create identities across each cloud provider are different. The goal of this enhancement is to bring some consistency to this process however users will still have to perform those manual actions, the reason for that is the cluster by design, does not have any privileged IAM actions and therefore these steps cannot be automated.

Reply:
changed wording.